core/views/directory.py

```diff
@@ -21,88 +21,19 @@
 
 @cors
 @cache_page(settings.DEFAULT_TTL_SECONDS)
-def newspapers(request, state=None, format='html'):
-    if state and state != "all_states":
-        state = unpack_url_path(state)
-        if state is None:
-            raise Http404
-        else:
-            state = state.title()
-    else:
-        state = request.GET.get("state", None)
-
-    language = request.GET.get('language', None)
-    if language:
-        language_display = models.Language.objects.get(code__contains=language).name
-    ethnicity = request.GET.get('ethnicity', None)
-
-    if not state and not language and not ethnicity:
-        page_title = 'All Digitized Newspapers'
-    else:
-        page_title = 'Results: Digitized Newspapers'
-
+def newspapers(request, city=None, format='html'):
+    page_title = 'Results: Digitized Newspapers'
```

Comment:
Since you dropped the queries (ethnicities, languages, etc), should this in fact be "All Digitized Newspapers" or something similar?  Results seems to imply that it is NOT all of them.